src/routes/docs/products/functions/execute/+page.markdoc

```diff
@@ -11,6 +11,42 @@ Appwrite Functions can be executed in several ways.
 Executions can be invoked through the Appwrite SDK and visiting its REST endpoint. Functions can also be triggered by events and scheduled executions.
 Here are all the different ways to consume your Appwrite Functions.
 
+# Execution modes {% #execution-modes %}
+
+Appwrite Functions support two execution modes: **synchronous** and **asynchronous**. Understanding the difference between these modes is important for building efficient applications.
+
+## Synchronous executions {% #synchronous-executions %}
+
+Synchronous executions are those where Appwrite makes the request to the function runtime synchronously and waits for the response. The client making the request will wait until the function completes and receives the response directly.
+
+Synchronous executions are created via:
+- The [Create execution](/docs/references/cloud/client-web/functions#createExecution) endpoint where the `async` parameter is `false`
+- Requests to custom or auto-generated [domains](/docs/products/functions/execute#domains)
+
+Synchronous executions:
+- Return response bodies and headers directly to the client
+- Have a **30-second timeout limit** to prevent long-running requests from blocking Appwrite workers
+- Are ideal for real-time operations where you need immediate response data
+- Work best for lightweight operations that complete quickly
```

Comment:
Ideally, all function executions should be lightweight and complete quickly. That's the suggestion for all serverless functions.

Reply:
Got it. I removed the line